apps/src/lib/util/analyticsUtils.js

```diff
@@ -0,0 +1,24 @@
+/* global appOptions */
+import {queryParams} from '@cdo/apps/code-studio/utils';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+export const reportTeacherReviewingStudentDslLevel = (
```

Comment:
Okay tbh even after a slack search I'm still not clear on what DSL means.

Reply:
Whoops I should've included this in my PR description. DSL means "domain specific language" and is essentially our levels that use our custom language to define them and inherit from [`DslDefined`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels/dsl_defined.rb).

I realize now that n `Widget` isn't actually a DSL level but I can't think of a better name. Maybe `reportTeacherReviewingStudentNonLabLevel`? Really rolls off the tongue 🙄